client/db/types.go

```diff
@@ -71,17 +74,26 @@ type AccountInfo struct {
 	FeeCoin   []byte
 	// Paid will be set on retrieval based on whether there is an AccountProof
 	// set.
-	Paid bool
+	Paid         bool
+	ClientPubKey *secp256k1.PublicKey
+	ID           account.AccountID
```

Comment:
Please make a note in the `AccountInfo` docs that the `ID` is meant to be unique and is used as the key for accounts in the DB.